packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -7,42 +7,65 @@
  * @prop --calcite-tile-border-color: Specifies the border color of the component.
  * @prop --calcite-tile-description-text-color: Specifies the description text color of the component.
  * @prop --calcite-tile-heading-text-color: Specifies the heading text color of the component.
+ * @prop --calcite-tile-icon-color: Specifies the icon color in the component.
+ * @prop --calcite-tile-selection-icon-color: Specifies the color of the selection icon in the component.
+ * @prop --calcite-tile-selection-icon-color-hover: Specifies the color of the selection icon in the component on hover.
+ * @prop --calcite-tile-shadow: Specifies the box-shadow of the component.
  */
 
+/** Internal variables
+  * --calcite-internal-tile-border-color
+  * --calcite-internal-tile-description-text-color
+  * --calcite-internal-tile-heading-text-color
+  * --calcite-internal-tile-icon-color
+  * --calcite-internal-tile-outline-color
+  * --calcite-internal-tile-selection-icon-color
+  * --calcite-internal-tile-shadow
+  */
+
 :host {
-  --calcite-tile-background-color: var(--calcite-color-foreground-1);
-  --calcite-tile-border-color: var(--calcite-color-border-2);
-  --calcite-tile-description-text-color: var(--calcite-color-text-3);
-  --calcite-tile-heading-text-color: var(--calcite-color-text-2);
-  --calcite-icon-color: var(--calcite-color-text-3);
+  --calcite-internal-tile-border-color: var(--calcite-tile-border-color, var(--calcite-color-border-2));
+  --calcite-internal-tile-description-text-color: var(
+    --calcite-tile-description-text-color,
+    var(--calcite-color-text-3)
+  );
+  --calcite-internal-tile-heading-text-color: var(--calcite-tile-heading-text-color, var(--calcite-color-text-2));
+  --calcite-internal-tile-icon-color: var(--calcite-color-text-3);
 
```

Comment:
set the default component color here.

```css
color: var(--calcite-tile-text-color, var(--calcite-color-text-3));
```

this will set the default for the component and cover the styles for the "calcite-icon" and "description" by default.